docs/getting_started.in.rst

```diff
@@ -144,9 +144,7 @@ demonstrate the usage of Open3D Python interface. See ``examples/python`` for
 all Python examples.
 
 .. note:: Open3D's Python tutorial utilizes some external packages: ``numpy``,
-    ``matplotlib``, ``opencv-python``. OpenCV is only used for reconstruction
```

Comment:
Removed missing shell script, as well as opencv-python (most users dont care about that) and also its easy to make this doc redundant when adding a new example.